apps/cowswap-frontend/src/modules/trade/containers/TradeWidget/TradeWidgetForm.tsx

```diff
@@ -90,6 +105,18 @@ export function TradeWidgetForm(props: TradeWidgetProps) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  const location = useLocation()
+  const isSwapMode = location.pathname.includes('/swap/')
```

Comment:
I think we have better ways to detect this, no?
I searched and apparently not.
Can you create a helper hook for this, then?

Something like:

```
function useIsSwapForm():boolean {
  return useLocation().pathname.includes('/swap/')
}
```

Reply:
Yeah didn't find a better way other than checking the current route. Addressed now.